sources/packages/forms/src/form-definitions/sfaa2025-26-ft.json

```diff
@@ -4089,27 +4089,27 @@
                     {
                       "label": "What are the daycare costs that you incur for your child(ren) age 11 years or under?",
                       "prefix": "$",
+                      "applyMaskOn": "change",
```

Comment:
Please include the thousands separator to make the value look similar to currency component. e.g. `100,000,99`